android/src/main/java/com/oktareactnative/OktaSdkBridgeModule.java

```diff
@@ -728,6 +735,14 @@ private int getRecalculatedRequestCodeForActivityResult(
         return initialRequestCode;
     }
 
+    private String getStackTraceString(Exception e) {
+        StringWriter sw = new StringWriter();
+        PrintWriter pw = new PrintWriter(sw);
+        e.printStackTrace(pw);
+        pw.flush();
+        return sw.toString();
+    }
```

Comment:
I got the source of this from Kotlin's definition of stackTraceToString function here: https://github.com/JetBrains/kotlin/blob/05896a42da04620a2e8ca35e952ff31c3c48aa9d/libraries/stdlib/jvm/src/kotlin/util/Exceptions.kt#L55